open_city_profile/settings.py

```diff
@@ -336,12 +337,29 @@ def _list_to_string(value):
 
 AUDIT_LOGGING_ENABLED = env.bool("AUDIT_LOGGING_ENABLED")
 AUDIT_LOG_USERNAME = env.bool("AUDIT_LOG_USERNAME")
+AUDIT_LOG_FILENAME = env("AUDIT_LOG_FILENAME")
+
+if AUDIT_LOG_FILENAME:
+    # Rotating log file containing only audit logs and
+    # intended for easy consumption by filebeat
+    _audit_log_handler = {
+        "level": "INFO",
+        "class": "logging.handlers.RotatingFileHandler",
+        "filename": AUDIT_LOG_FILENAME,
+        "maxBytes": 100000000,  # 100 MiB
+        "backupCount": 1,  # Keep one previous version
```

Comment:
These comments are pretty useless. You could write the `maxBytes` value as `100_000_000` to improve readability.